static/sass/ds/6-components/_appeal.scss

```diff
@@ -0,0 +1,21 @@
+.c-appeal {
+  list-style: none;
+  padding-bottom: $size-xxl;
+
+  @include mq($until: bp-l) {
+    padding-top: $size-xxl;
+  }
+
+  &__item {
+    align-items: center;
+    display: flex;
+
+    > .c-icon {
+      margin-right: $size-xs;
+    }
```

Comment:
To avoid that extra level of specificity, it seems like you wouldn't need the `>` direct child selector. Unless we foresee more deeply nested icons.

To take that a step further, you could even say `c-appeal__icon` would almost be the real way to target that and keep it low specificity, but I'm not sure if the icon component make that harder to add.

Reply:
I removed the direct-child selector. I typically add it by default because I've been burned way too many times in the past by accidentally styling deep descendants ... but I also totally get why it's probably not best practice to include it unless needed.

Re: your second point ... I'm torn. I think your way is probably the "official" BEM way to do it, but I also don't like how that way make it seem like the `.c-appeal__icon` element is a blank slate when in fact it has stying baked in via `.c-icon`. I think I prefer being explicit and letting my CSS say, "Be careful styling that nested icon!"

Reply:
Makes sense to me! Child combinators as it turns out, are the same level of specificity, so not a deal breaker to use that sometimes. I know what you mean about getting burned by inheritance, but yeah I like only using those when the designation is absolutely needed.